image-layout.md

```diff
@@ -0,0 +1,47 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable objects and human significant references (refs) 
+This layout MAY be used in a variety of different transport mechanisms from archive formats (e.g. tar, zip) or used in shared filesystem environments (e.g. nfs) or in networked file fetching systems (e.g. http, ftp, rsync).
```

Comment:
nit: the “from … or … or” phrasing is a bit awkward.  How about “transport mechanisms, including archive formats (…), shared filesystems (…), and networked transport protocols (…).  Although at this level, the distinction between NFS and HTTP is pretty irrelevant, so it's probably not worth looking to hard for good bucket names.